### DISCUSSION TITLE
Checking in for liteshell

### DISCUSSION THREAD

**Discussion by thenoblet** [codemindshub/liteshell#2] [General]
You are finally here to collaborate on `liteshell (lsh)` .

Introduce yourself and tell us what you aim to achieve.

**Comment by Michael-Sylvester** [comment]
I'm Michael. 
I'm here to get a better understanding of what I can do by collaborating with everyone else and sharing ideas

**Reply by thenoblet** [reply]
Welcome, Mentor Michael...

**Comment by KeaMohlala** [comment]
I am hoping that this liteshell will help me better understand the simple shell project because I am feeling overwhelmed.

**Reply by thenoblet** [reply]
Perfect, Kea. Looking forward to a great collaboration

**Comment by Gbayilo** [comment]
Hi guys! I.m Gbayilo. I look forward to learning how to collaborate effectively with everyone to achieve the set goal of building a liteshell.

**Reply by thenoblet** [reply]
Glad to have you here, ``Gbayilo.``

**Comment by tijesus** [comment]
Hi guys, i'm Gabriel. Looking forward to learning and building with everyone cos I'm feeling stuck on the project and coding generally

**Comment by akoredera** [comment]
I am Ridwanullahi, and I am looking forward to learning new things, share ideas regarding the ongoing project and improving my knowledge in the use of github

**Reply by thenoblet** [reply]
Welcome, Rid.

**Comment by OmondiHillaryA1997** [comment]
Am Hillary. Excited to be here and looking forward not only for this collaboration but more. Aiming to bring out, the best of me through the help of this able team of Mentors.

Hope some day, we will look back and appreciate the Process.
Grateful to be here.

**Comment by gr8justus** [comment]
Hello everyone,

I'm Oluwatobi from Lagos, Nigeria. I am excited to share my insights and learn from each of you throughout this project and beyond. I am committed to contributing to the success of our collaboration by ensuring tasks are completed.

I feel privileged to be a part of this community.

Looking forward to our collective success!

Best regards,
Oluwatobi
